tests/Dfc.CourseDirectory.Testing/ExecuteImmediatelyBackgroundWorkScheduler.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Threading.Tasks;
+using Dfc.CourseDirectory.Core.BackgroundWorkers;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Dfc.CourseDirectory.Testing
+{
+    public class ExecuteImmediatelyBackgroundWorkScheduler : IBackgroundWorkScheduler
```

Comment:
I can't see where this is being used...

Reply:
I needed this as part of next PR for testing but it felt like it made more sense in this PR